### PR TITLE
[#27] Part 1

### PR DESCRIPTION
**Pull request by TejasSC**

### CODE REVIEW

File: fix-point/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.stack-work/
```
Comment (ChShersh) [inline fix-point/.gitignore]: This `.gitignore` file is not needed. It's enough to have single `.gitignore` per repository.

File: fix-point/ChangeLog.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Changelog for fix-point
+
+## Unreleased changes
```
Comment (ChShersh) [inline fix-point/ChangeLog.md]: I think you can remove changelog file as well because probably nobody will maintain it.

File: fix-point/LICENSE [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Copyright Author name here (c) 2018
```
Comment (ChShersh) [inline fix-point/LICENSE]: There's top-level LICENSE in project so you can remove LICENSE inside `fix-point`

File: fix-point/Setup.hs [hunk 4]
```diff
@@ -0,0 +1,2 @@
+import Distribution.Simple
+main = defaultMain
```
Comment (ChShersh) [inline fix-point/Setup.hs]: `Setup.hs` file is not needed.

File: fix-point/stack.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+# resolver: nightly-2015-09-21
+# resolver: ghc-7.10.2
+# resolver: ghcjs-0.1.0_ghc-7.10.2
+#
+# The location of a snapshot can be provided as a file or url. Stack assumes
+# a snapshot provided as a file might change, whereas a url resource does not.
+#
+# resolver: ./custom-snapshot.yaml
+# resolver: https://example.com/snapshots/2018-01-01.yaml
+resolver: lts-11.16
```
Comment (ChShersh) [inline fix-point/stack.yaml]: You can remove every line in this file except this one.

File: testFile.hs [hunk 6]
```diff
@@ -0,0 +1,8 @@
+module ParseStuff where 
+
+parseStuff :: String -> String
+parseStuff s = case s of
```
Comment (ChShersh) [inline testFile.hs]: What is the purpose of this file?

File: fix-point/stack.yaml (hunk 5 above)
Comment (ChShersh) [reply fix-point/stack.yaml]: Still waiting for cleaning this file

File: fix-point/fix-point.cabal [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+bug-reports:    https://github.com/githubuser/fix-point/issues
+author:         Author name here
+maintainer:     example@example.com
+copyright:      2018 Author name here
+license:        BSD3
+license-file:   LICENSE
+build-type:     Simple
+cabal-version:  >= 1.10
+extra-source-files:
+    ChangeLog.md
```
Comment (ChShersh) [inline fix-point/fix-point.cabal]: Since you don't have changelog this line can be removed

File: fix-point/fix-point.cabal [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+name:           fix-point
+version:        0.1.0.0
+description:    Please see the README on GitHub at <https://github.com/githubuser/fix-point#readme>
+homepage:       https://github.com/githubuser/fix-point#readme
+bug-reports:    https://github.com/githubuser/fix-point/issues
+author:         Author name here
+maintainer:     example@example.com
+copyright:      2018 Author name here
+license:        BSD3
+license-file:   LICENSE
```
Comment (ChShersh) [inline fix-point/fix-point.cabal]: Did you try to build this package? Is it buildable?

**Comment by ChShersh** [comment]
Also, you don't need to create separate commit for deleting each file. But this PR will be squashed and merged anyways.

File: fix-point/fix-point.cabal (hunk 8 above)
Comment (TejasSC) [reply fix-point/fix-point.cabal]: Package is buildable and executable without error.

File: testFile.hs (hunk 6 above)
Comment (TejasSC) [reply testFile.hs]: It was a test file I tried to commit and push to the master branch before I realised I shouldn't do that.

**Pull request merged**